packages/clerk-js/src/v4/icons/index.ts

```diff
@@ -6,6 +6,8 @@
  */
 export { default as ArrowLeftIcon } from './arrow-left.svg';
 export { default as ArrowRightIcon } from './arrow-right.svg';
+export { default as AuthenticatorApp } from './authenticator-app.svg';
```

Comment:
We already have an icon named AuthApp that's the same, shall we use that instead? This one looks greyish so I guess it has a hardcoded color set within its definition

Reply:
Replacing with `AuthApp`